dashboard/maintainers.txt

```diff
@@ -1,2 +0,0 @@
-lskibinski@elifesciences.org
```

Comment:
did this file need to be moved to the root of the project?

Reply:
Yup, good spot